### PR TITLE
fix(inwonersaantallen): custom municipality matching

### PR DESCRIPTION
**Pull request by rikdepeuter**

### CODE REVIEW

File: src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Brecht.cs [hunk 1]
```diff
@@ -17,6 +17,19 @@ public Brecht(string nisCode, string sourceHouseNumber, NationalRegistryIndex in
                 IndexSourceValue.StartsWith("GV", StringComparison.InvariantCultureIgnoreCase) && IsNumeric(IndexSourceValue[2..])
                 ||
                 IndexSourceValue[0] == '0' && IsLetter(IndexSourceValue[3])
+                ||
```
Comment (jvandaal) [inline src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Brecht.cs:20]: Doesn't match with GetValues()?

File: src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Herentals.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    public class Herentals : MunicipalityHouseNumberBoxNumbersBase
+    {
+        public Herentals(string nisCode, string sourceHouseNumber, NationalRegistryIndex index) : base(nisCode, sourceHouseNumber, index)
+        { }
+
+        public override bool IsMatch() =>
+            NisCode == "13011" &&
+            (
+                IndexSourceValue.Length == 4 && IsLetter(IndexSourceValue[0]) && IndexSourceValue[1..] == "000"
```
Comment (jvandaal) [inline src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Herentals.cs:14]: IndexSourceValue is always 4 characters long.
image

File: src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Mortsel.cs [hunk 3]
```diff
@@ -23,6 +23,14 @@ public override IList<HouseNumberWithBoxNumber> GetValues()
...
                         HouseNumberSourceValue,
                         "glv"
+                    ),
+                    new HouseNumberWithBoxNumber(
+                        HouseNumberSourceValue,
+                        "glvl"
+                    ),
+                    new HouseNumberWithBoxNumber(
+                        HouseNumberSourceValue,
+                        "GLVL"
```
Comment (jvandaal) [inline src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Mortsel.cs:33]: When matching addresses, we ignore cases, see https://github.com/Informatievlaanderen/basisregisters-integration-db/blob/main/src/Basisregisters.IntegrationDb.NationalRegistry/AddressMatching/AddressMatchRunner.cs#L66

**Review by jvandaal — APPROVED** [review]

File: src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Mortsel.cs (hunk 3 above)
Comment (rikdepeuter) [reply src/Basisregisters.IntegrationDb.NationalRegistry/Model/HouseNumberBoxNumberImplementations/Municipalities/Mortsel.cs]: Fixed

**Review by jvandaal — APPROVED** [review]

**Pull request merged**